mirakuru/http.py

```diff
@@ -50,16 +53,21 @@ def __init__(self, command, url, **kwargs):
         """
         An :func:`urlparse.urlparse` representation of an url.
 
-        It'll be used to check process status on."""
+        It'll be used to check process status on.
+        """
+
+        port = self.url.port
```

Comment:
This 'idiom' sometimes breaks ('falsy' values), therefore it's not clean. Also that's not what `or` is for.